test/Makefile

```diff
@@ -51,7 +51,17 @@ $(INITRAMFS)/lib/x86_64-linux-gnu: | $(VDSO_LIB)
 
 $(VDSO_LIB): | $(VDSO_DIR)
 	@# TODO: use a custom compiled vdso.so file in the future.
-	@wget https://raw.githubusercontent.com/asterinas/linux_vdso/2a6d2db/vdso64.so -O $@ 
+	@if [ ! -d "/tmp" ]; then \
+		sudo mkdir /tmp && \
+		sudo chmod 1777 /tmp || { \
```

Comment:
I don't understand why we can't assume that `/tmp` exists. Isn't it a standard directory on any Linux machine?

Reply:
I believe unix-like systems have `/tmp` unless the administrator customizes the system, so it makes sense to remove the check.